actioncable/lib/rails/generators/channel/templates/assets/channel.js

```diff
@@ -0,0 +1,18 @@
+App.<%= class_name.underscore %> = App.cable.subscriptions.create("<%= class_name %>Channel", {
+  connected: function() {
+    // Called when the subscription is ready for use on the server
+  },
+
+  disconnected: function() {
+    // Called when the subscription has been terminated by the server
+  },
+
+  received: function(data) {
+    // Called when there's incoming data on the websocket for this channel
+  }<%= actions.count > 0 ? ",\n" : '' %>
```

Comment:
This is also good but would like to keep `actions` out of the `received` function. Going with `actions.any?` approach.